web/client/epics/annotations.js

```diff
@@ -619,14 +623,14 @@ export default {
             // TODO update selected feature in editing features
 
             let selectedIndex = findIndex(feature.features, (f) => f.properties.id === selected.properties.id);
-            if (selected.properties.isValidFeature || selected.geometry.type === "LineString" || selected.geometry.type === "MultiPoint" || selected.geometry.type === "Polygon") {
+            if (selected.properties.isValidFeature || includes(featureTypes, selected.geometry.type)) {
```

Comment:
in this way you are changing the number of types that the previous condition was setup
for example here it will be included Point where before it wasn't
is it intended?

Reply:
Yes, this is intended. This specific change resolves the issue when new "Point" geometry is created when user clicks on the map while currently selected Text or Marker is invalid. Both Marker end text are "Point" feature with custom properties, and old condition was forcing creation of a new feature if selected is invalid. With new condition invalid feature is not being filtered out and eventually coordinates of the point are being updated after click.